city_scrapers/spiders/wayne_state_university.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.spiders import CityScrapersSpider
+import re
+
+
+class WayneStateUniversitySpider(CityScrapersSpider):
+    name = "wayne_state_university"
+    agency = "Wayne State University"
+    timezone = "America/Chicago"
+    allowed_domains = ["bog.wayne.edu"]
+    start_urls = ["http://bog.wayne.edu/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        
+        #one large all encompassing meeting
+        meeting_dates = response.xpath('//*[@id="content"]/div[1]/div/h2').re('((?:January|Feburary|March|April|May|June|July|August|October|September|November|December)\s+\d{1,2},\s+\d{4})')
+        for item in meeting_dates:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification= BOARD,
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board of Governors and Committee Meeting"
+
+    
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return BOARD 
+  
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date = item
+        time = response.xpath('//*[@id="content"]/div[1]/div/ul').re('(\d+:\d{1,2}\s(?:a.m|p.m))')
+        date_time_start = date + " at " + time[0]
+        return data_time_start
+
+    
+    def _parse_end(self, item):
+        """Parse start datetime as a naive datetime object."""
+        time = response.xpath('//*[@id="content"]/div[1]/div/ul').re('(\d+:\d{1,2}\s(?:a.m|p.m))')
+        date_time_start = item
+        date_time_end = date + " at " + time[-1]
+
+        return date_time_end
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return "Unless noted otherwise, the time schedule for meetings will be as follows"
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False #no all day meetings to be seen
+
+    def _parse_location(self, item):
```

Comment:
You'll want to use a `_validate_location` function here to verify that it hasn't changed. Here's one example: https://github.com/City-Bureau/city-scrapers/blob/c23f1fd17b1ced1c01dff295c8cdf1ca191f0244/city_scrapers/spiders/chi_ssa_34.py#L58